### DISCUSSION TITLE
Highlight OP of comment in a different color as compared to other comments

### DISCUSSION THREAD

**Discussion by Humorist2601** [Docile-Alligator/Infinity-For-Reddit#991] [Ideas]
In Slide for reddit, this is implemented by keeping the color of the comment depth indicator purple on the OPs comment.

**Comment by glibg10b** [comment]
Their username is a different color and there is a microphone next to it

**Comment by Humorist2601** [comment]
The comment OP, not the post OP.

**Comment by jake770321** [comment]
Would love if this feature was implemented like it is on slide. Ideally with the option to set a custom colour for the comment level indicator for any comments by the comment OP
